src/Package/Impl/DataInspect/DataImport/ImportDataWindow.xaml.cs

```diff
@@ -57,9 +57,9 @@ public partial class ImportDataWindow : DialogWindow {
 
         public IDictionary<string, int> Encodings { get; } = new Dictionary<string, int>();
 
-        public IDictionary<string, string> RowNames { get; } = new Dictionary<string, string> {
+        public IDictionary<string, int?> RowNames { get; } = new Dictionary<string, int?> {
```

Comment:
This is still controlling what we are passing to `row.names` parameter of `read.csv`.